server/src/main/java/org/elasticsearch/rest/BaseRestHandler.java

```diff
@@ -51,12 +53,13 @@
  * are copied, but a selected few. It is possible to control what headers are copied over by returning them in
  * {@link ActionPlugin#getRestHeaders()}.
  */
-public abstract class BaseRestHandler extends AbstractComponent implements RestHandler {
+public abstract class BaseRestHandler implements RestHandler {
 
     public static final Setting<Boolean> MULTI_ALLOW_EXPLICIT_INDEX =
         Setting.boolSetting("rest.action.multi.allow_explicit_index", true, Property.NodeScope);
 
     private final LongAdder usageCount = new LongAdder();
+    private static final Logger logger = LogManager.getLogger(BaseRestHandler.class);
```

Comment:
This logger appears to be unused.

Reply:
I was thinking of having something like:
```
@Deprecated
protected static Logger logger = LogMangaer.getLogger(getClass());
```

in 6.x and master and then removing it entirely from master in a follow up change.